prow/plugins/config.go

```diff
@@ -50,6 +51,10 @@ type Configuration struct {
 	// Owners contains configuration related to handling OWNERS files.
 	Owners Owners `json:"owners,omitempty"`
 
+	// GithubOptions allows users to control how plugins interact with
+	// git, github, and github API's.
+	GithubOptions GithubOptions `json:"github,omitempty"`
```

Comment:
This means we would have to double the config and we no longer have one source of truth. I'm not sure that is a good idea

Reply:
True... how about we just rely on github.Client instead?